designsafe/static/scripts/ng-designsafe/providers/notifications-provider.js

```diff
@@ -148,16 +148,16 @@ function NotificationService(
         });
 
         const toastViewLink = renderLink(msg);
-        if (typeof toastViewLink !== 'undefined') {
```

Comment:
Is this code relevant to v3 ?

Reply:
It's actually relevant to the old code, but this code is still "live" on the v3 page and was causing duplicate toast notifications from this old code. We'll need to rip this all out in time, but short term this disables the old code from producing notifications